experimental/LinearQuotients/src/misc.jl

```diff
@@ -1,27 +1,38 @@
-# Try to 'update' the base_ring of G
-function map_entries(K::Field, G::MatrixGroup)
-  g = dense_matrix_type(K)[]
-  for h in gens(G)
-    push!(g, map_entries(K, h.elm))
-  end
-  return matrix_group(g)
-end
+@doc raw"""
+    is_reflection(g::MatrixGroupElem)
 
+Return `true` if `g` is a reflection, `false` otherwise. By a reflection, we mean
+a matrix with fixed space of codimension 1.
```

Comment:
I am not sure that's a generally accepted definition for reflection; I'd also expect the final eigenvalue to be -1...

Reply:
Should I change it to 'pseudoreflection'?

Reply:
For what it's worth, in Lehrer--Taylor "Unitary reflection groups" it is called reflection. But I can't argue against the authority of Bourbaki :)